src/Test.FunctionalTests.BinSkim.Driver/AnalyzeCommandTests.cs

```diff
@@ -65,7 +65,11 @@ public void AnalyzeCommand_ReadSarifLog_ShouldBeAbleToReadCurrent()
         [Obsolete]
         public void AnalyzeCommand_Hashes_ShouldUpdateDataToInsert()
```

Comment:
>AnalyzeCommand_Hashes_ShouldUpdateDataToInsert [](http://example.com/codeflow?start=20&length=46)

This test were failing in the unix pipelines. With the changes below, i was able to make it pass.